### PR TITLE
Docs/add ECS migration section to readme

### PR DESCRIPTION
**Pull request by chillier-ch**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: README.md [hunk 1]
```diff
@@ -33,3 +33,34 @@ Path | Method | Description
...
+- Ensure the terraform runner local plan executes without issues. For information on terraform runners please see [here](https://companieshouse.atlassian.net/wiki/spaces/DEVOPS/pages/1694236886/Terraform+Runner+Quickstart)
+- If you encounter any issues or have questions, reach out to the team on the **#platform** slack channel.
+
+### Vault Configuration Updates
+- Any secrets required for this service will be stored in Vault. For any updates to the Vault configuration, please consult with the **#platform** team and submit a workflow request.
+
+### Useful Links
+- [ECS service config dev repository](https://github.com/companieshouse/ecs-service-configs-dev)
+- [ECS service config production repository](https://github.com/companieshouse/ecs-service-configs-production)
+- [Terraform runners quickstart docs](https://companieshouse.atlassian.net/wiki/spaces/DEVOPS/pages/1694236886/Terraform+Runner+Quickstart)
```
[the remark this answers is not in the document]
Comment (chillier-ch) [reply README.md]: Have removed that duplicated link now 👍

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Pull request merged**